diff --git a/node_modules/next/dist/esm/server/app-render/create-component-styles-and-scripts.js b/node_modules/next/dist/esm/server/app-render/create-component-styles-and-scripts.js
index 2d3806b..6a90602 100644
--- a/node_modules/next/dist/esm/server/app-render/create-component-styles-and-scripts.js
+++ b/node_modules/next/dist/esm/server/app-render/create-component-styles-and-scripts.js
@@ -10,7 +10,8 @@ export async function createComponentStylesAndScripts({ filePath, getComponent,
     const scripts = jsHrefs ? jsHrefs.map((href, index)=>createElement('script', {
             src: `${ctx.assetPrefix}/_next/${encodeURIPath(href)}${getAssetQueryString(ctx, true)}`,
             async: true,
-            key: `script-${index}`
+            key: `script-${index}`,
+            nonce: ctx.nonce
         })) : null;
     const Comp = interopDefault(await getComponent());
     return [
diff --git a/node_modules/next/dist/server/app-render/create-component-styles-and-scripts.js b/node_modules/next/dist/server/app-render/create-component-styles-and-scripts.js
index ccfd371..bc4c48a 100644
--- a/node_modules/next/dist/server/app-render/create-component-styles-and-scripts.js
+++ b/node_modules/next/dist/server/app-render/create-component-styles-and-scripts.js
@@ -20,7 +20,8 @@ async function createComponentStylesAndScripts({ filePath, getComponent, injecte
     const scripts = jsHrefs ? jsHrefs.map((href, index)=>createElement('script', {
             src: `${ctx.assetPrefix}/_next/${(0, _encodeuripath.encodeURIPath)(href)}${(0, _getassetquerystring.getAssetQueryString)(ctx, true)}`,
             async: true,
-            key: `script-${index}`
+            key: `script-${index}`,
+            nonce: ctx.nonce
         })) : null;
     const Comp = (0, _interopdefault.interopDefault)(await getComponent());
     return [
